commands/command_filter_process.go

```diff
@@ -203,10 +203,14 @@ func filterCommand(cmd *cobra.Command, args []string) {
 	}
 
 	if len(malformed) > 0 {
-		fmt.Fprintf(os.Stderr, "Encountered %d file(s) that should have been Git LFS pointers, but they're Git files:\n", len(malformed))
+		fmt.Fprintf(os.Stderr, "Encountered %d file(s) that should have been Git LFS pointers, but weren't:\n", len(malformed))
+
 		for _, m := range malformed {
 			fmt.Fprintf(os.Stderr, "\t%s\n", m)
 		}
+
+		fmt.Fprintf(os.Stderr, "\Type: `git lfs ls-files` to see which files are in the Git LFS index.\n");
+		fmt.Fprintf(os.Stderr, "\See: `git lfs help migrate` for migrating files to the Git LFS index.\n");
```

Comment:
I think these two lines have syntax errors.

Also, since this is a warning message, we should try to be fairly brief.  I think we could just say, “See "git lfs migrate --help" for how to migrate these files to Git LFS.” I'm not sure we need the previous line, and there isn't really a Git LFS index separate from the Git index.